website/docs/cloud-docs/agents/telemetry.mdx

```diff
@@ -4,40 +4,104 @@ page_title: Telemetry - Terraform Cloud Agents - Terraform Cloud and Terraform E
+[OpenTelemetry](https://opentelemetry.io/) protocol. The agent is a simple
+producer of telemetry data, and relies on an OpenTelemetry Collector to perform
+aggregation, post-processing, and exporting of telemetry data to upstream
+technology and/or vendors. OpenTelemetry is an extensible telemetry protocol and
+open source platform, which enables the Terraform Cloud Agent to remain
+vendor-agnostic while producing high-value telemetry data for a wide array
+of platforms.
 
-Optionally, you can pass the `-otlp-cert-file` or `TFC_AGENT_OTLP_CERT_FILE`. The agent will use a certificate at the path supplied to encrypt the client connection to the OpenTelemetry collector. When omitted, client connections are not secured.
+![telemetry architecture diagram](/img/telemetry-arch.svg)
 
-## Metrics
+## Agent Configuration
+
+To configure your agent to emit telemetry data, you must include the
+`-otlp-address` flag or `TFC_AGENT_OTLP_ADDRESS` environment variable. This
+should be set to the host:port address of an [OpenTelemetry
+collector](https://opentelemetry.io/docs/concepts/data-collection/). This
+address should be a GRPC server running an OLTP collector.
+
+Optionally, you can pass the `-otlp-cert-file` or `TFC_AGENT_OTLP_CERT_FILE`.
+The agent will use a certificate at the path supplied to encrypt the client
+connection to the OpenTelemetry collector. When omitted, client connections are
+not secured.
+
+## OpenTelemetry Collector
+
+The OpenTelemetry Collector has a pluggable architecture. This opens up the
+ecosystem and makes it possible to leverage a wide array of telemetry platforms
+and technologies. The OpenTelemetry project distributes the collector in two
+flavors: `core` and `contrib`. For most uses, the `contrib` distribution will
+be most useful, as it includes a wide array of integrations for specific
+telemetry platforms and technologies which are not available in the `core`
+distribution. The `contrib` distribution is available as a pre-packaged
+Docker container labeled
+[otel/opentelemetry-collector-contrib](https://hub.docker.com/r/otel/opentelemetry-collector-contrib).
+
+## Compatibility
+
+Each released version of the Terraform Cloud Agent ships with a specific version
+of the OpenTelemetry client libraries built in. As these libraries and
+implementations change, it is possible that certain versions of the agent will
+become incompatible with newer versions of the OpenTelemetry collector. To
+track version compatibility, the table below will be updated whenever these
+version constraints become known.
 
-An example of the metric names that the agent will emit is `tfc-agent.terraform.plan-json.generate.bytes`. Breaking down those sections:
+| Terraform Cloud Agent version | Compatible OpenTelemetry Collector version |
+| ----------------------------- | ------------------------------------------ |
+| All versions                  | <= 0.48.0                                  |
```

Comment:
although 0.48.0 is _technically_ compatible with the current libraries used, i remember that not all data was making it to Datadog, or GCP when attempting to utilize this version, and it wasn't until i made it down to `0.42.0`, the version we use internally, that i was able to see all telemetry making it successfully into DD at the least. We may want to change this to 0.42.0, as it's the best known version to work with how we have things instrumented.

Reply:
Let's go with 0.42.0 then 👍